lib/CheckboxItem/CheckboxItemAccessibilitySupport.js

```diff
@@ -0,0 +1,30 @@
+var
+	kind = require('enyo/kind');
+
+/**
+* @name CheckboxItemAccessibilityMixin
+* @mixin
+*/
+module.exports = {
+
+	/**
+	* @private
+	*/
+	observers: [
+		{method: 'updateAccessibilityAttributes', path: ['checked']}
+	],
+
+	/**
+	* @private
+	*/
+	updateAccessibilityAttributes: kind.inherit(function (sup) {
+		return function (props) {
+			var enabled = !this.accessibilityDisabled;
+			sup.apply(this, arguments);
+			this.setAttribute('role', enabled ? 'checkbox' : null);
+			this.setAttribute('tabindex', enabled ? 0 : null);
+			this.setAttribute('aria-checked', enabled ? String(this.checked) : null);
+			this.setAttribute('aria-labelledby', enabled ? this.getId() : null);
```

Comment:
We use aria-labelledby to read child components traversely. If spotlight focused component has some components as child, we add aria-labelledby to this spotlight focused component. In CheckboxItem, your opinion also is right but we thought that aria-labelledby should add to spotlight focused components in moonstone.